docker/README.md

```diff
@@ -0,0 +1,29 @@
+# fMRI in docker
+
+## Usage
+
+```shell
+docker run -it yupri/fmri:fsl-latest
+```
```

Comment:
Didn't upload to Docker Hub yet. If you want to play around -- follow step 2 and further below.